js/bitget.js

```diff
@@ -3095,7 +3092,190 @@ module.exports = class bitget extends Exchange {
         //
         const data = this.safeValue (response, 'data');
         const orderList = this.safeValue (data, 'orderList', data);
-        return this.parseOrders (orderList, market, since, limit);
+        const result = [];
+        for (let i = 0; i < orderList.length; i++) {
+            const entry = orderList[i];
+            const status = this.parseOrderStatus (this.safeString2 (entry, 'state', 'status'));
+            if (status === 'closed') {
+                result.push (entry);
+            }
+        }
+        return this.parseOrders (result, market, since, limit);
+    }
+
+    async fetchCanceledOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
isn't this an emulated `fetchCanceledOrders` using the exact same endpoint as `fetchClosedOrders` ? if so we should not duplicate the logic but call it here and do only the filtering in the end

Reply:
sounds good I pushed some changes